### ISSUE TITLE
Ajout de marges css au niveau du menu

### ISSUE THREAD

**Issue by johannchopin** [laowantong/d3_stuff#2]
![image](https://user-images.githubusercontent.com/31794680/36620344-d7efa9bc-18f2-11e8-8ae7-686d9d94b50f.png)

Pourquoi ne pas ajouter des marges comme sur la photo au niveau du menu de gauche ? Ce n'est qu'une question de goût après 😄 Il suffit juste de rajouter ces quelques lignes au script css :
`#menu{ margin: 0 20px; }`

P.S: C'est bien une issues comme celle là qu'il faut faire pour ce genre de proposition non ?

**Comment by laowantong** [comment]
Oui, on va faire comme ça. Ceci dit, mieux vaut restreindre une copie d'écran à la zone concernée. Ici, en fait, je ne vois pas la différence avec l'état actuel. Chez toi il n'y a pas de marge sans ça?

**Comment by johannchopin** [comment]
Ah peut être que c'est juste chez moi que ça ne fonctionne pas car de base je vois ceci :
![capture](https://user-images.githubusercontent.com/31794680/36629716-5379cb46-195a-11e8-8c49-dc2aab80eeb5.jpg)

La capture d'écran provient de Firefox Quantum sous Windows 10

**Comment by laowantong** [comment]
Même problème en effet chez moi sous Firefox, mais pas sous Chrome ni Safari. PR bienvenue :smile:

**Comment by johannchopin** [comment]
Ajouter la simple ligne css écrite plus haut permettrait de régler ce bug sur chaque navigateurs 😉

**Comment by laowantong** [comment]
D'accord, super! Peux-tu faire une branche et une _pull request_ pour t'entraîner?
